### PR TITLE
Aws lambda support

### PR DESCRIPTION
**Pull request by mattonem**
Other commands to come.
doc: https://docs.aws.amazon.com/lambda/latest/dg/API_Operations.html

### CODE REVIEW

**Comment by newapplesho** [comment]
@mattonem 
Can you write "how to use" ?
https://github.com/newapplesho/aws-sdk-smalltalk/wiki

File: pharo-repository/AWS-Lambda/AWSLambdaConfig.class.st [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	default := nil.
+]
+
+{ #category : #defaults }
+AWSLambdaConfig >> defaultServiceName [
+	^ 'lambda'
+]
+
+{ #category : #accessing }
+AWSLambdaConfig >> hostUrl [
```
Comment (newapplesho) [inline pharo-repository/AWS-Lambda/AWSLambdaConfig.class.st]: https://github.com/newapplesho/aws-sdk-smalltalk/blob/master/pharo-repository/AWS-Core/AWSConfig.class.st#L87

Comment (mattonem) [reply pharo-repository/AWS-Lambda/AWSLambdaConfig.class.st]: indeed. 
I removed it.

**Comment by mattonem** [comment]
"how to use" available here: https://github.com/newapplesho/aws-sdk-smalltalk/wiki/Lambda

**Pull request merged**